internal/codeintel/autoindexing/index_records.go

```diff
@@ -47,12 +47,14 @@ func makeExplicitConfigurationFactory(configuration string) configurationFactory
 			return nil, false, nil
 		}
 
+		logger := log.Scoped("makeExplicitConfigurationFactory", "")
```

Comment:
```suggestion
		logger := log.Scoped("explicitConfigurationFactory", "")
```

`make` is the verb in the constructor, it's not needed here

Reply:
also, let's move this to between line 44 and line 45 so that the logger isn't created repeatedly